pkg/storage/read_config.go

```diff
@@ -0,0 +1,49 @@
+package storage
+
+import (
+	"context"
+)
+
+// PostgresReadConfig contains the Postgres database configuration arguments for the ReadReplica
+type PostgresReadConfig struct {
+	Hostname     string
+	Port         string
```

Comment:
question: why is port a string?

Reply:
This is the same with existing PostgresConfig. 
I will try to change it to UInt16 next time I touch it and see what breaks.

Reply:
Done